boards/arm/nucleo_wl55jc/Kconfig.defconfig

```diff
@@ -9,6 +9,6 @@ config BOARD
 	default "nucleo_wl55jc"
 
 config SYS_CLOCK_TICKS_PER_SEC
-	default 4000 if STM32_LPTIM_TIMER
+	default 4096 if STM32_LPTIM_TIMER
```

Comment:
I see. Given that #64077 will set LSE as default clock for LPTIM, we still need the changes in `nucleo_wl55jc.dts` in this PR, but not the `SYS_CLOCK_TICKS_PER_SEC` hack.

I you want I can remove the second commit from this PR (so that the PR is only about the backup memory).

I can defer the second commit f3a5de801b51dd38bca70891f80ebaa28add06d8 to another PR, pending #64067 merge.

Reply:
> I can defer the second commit https://github.com/zephyrproject-rtos/zephyr/commit/f3a5de801b51dd38bca70891f80ebaa28add06d8 to another PR, pending https://github.com/zephyrproject-rtos/zephyr/pull/64067 merge.

Not blocking but this is better practice in general indeed